docs/js/features/odata/use-generated-client.md

```diff
@@ -0,0 +1,389 @@
+
+
+```ts
+BusinessPartner.requestBuilder()
+    .getAll()
+    .select(
+        BusinessPartner.FIRST_NAME,
+        BusinessPartner.LAST_NAME,
+        BusinessPartner.TO_BUSINESS_PARTNER_ADDRESS
+    )
+    .execute(destination);
+```
+
+The above translates to the following query parameters:
+
+```sql
+$select=FirstName,LastName,to_BusinessPartnerAddress
+```
+
+<!-- OData v4
+```sql
+$select=FirstName,LastName&$expand=to_BusinessPartnerAddress
+```
+-->
+
+One can also select properties of the expanded object:
+
+```ts
+BusinessPartner.requestBuilder()
+    .getAll()
+    .select(
+        BusinessPartner.FIRST_NAME,
+        BusinessPartner.TO_BUSINESS_PARTNER_ADDRESS.select(
+            BusinessPartnerAddress.ADDRESS_ID,
+            BusinessPartnerAddress.CITY_CODE
+        )
+    )
+    .execute(destination);
+```
+
+The above translates to the following query parameters:
+
+```sql
+$select=FirstName,to_BusinessPartnerAddress/AddressID,to_BusinessPartnerAddress/CityCode&$expand=to_BusinessPartnerAddress
+```
+
+### Filter
+
+When operating on a collection of entities the API offers `filter( ... )` on the builders. It directly corresponds to the `$filter` parameter of the request. Filters are also build via the static property fields on entities.
+
+The following example:
+
+```ts
+/*
+Get all business partners that either:
+  - Have first name 'Alice' but not last name 'Bob'
+  - Or have first name 'Mallory'
+*/
+BusinessPartner.requestBuilder()
+    .getAll()
+    .filter(
+        or(
+            and(
+                BusinessPartner.FIRST_NAME.equals('Alice'),
+                BusinessPartner.LAST_NAME.notEquals('Bob')
+            ),
+            BusinessPartner.FIRST_NAME.equals('Mallory')
+        )
+    )
+    .execute(destination);
+```
+
+Will translate to this filter parameter:
+```sql
+$filter=(((FirstName eq 'Alice') and (LastName ne 'Bob')) or (FirstName eq 'Mallory'))
+```
+
+Take note of the order of `and` and `or`. As `or` is invoked on the result of `and` it will form the outer expression while `and` is an inner expression in the first branch of `or`.
+
+It's also possible to add multiple filters in the same filter function without using `and`, as they will be concatenated. This example:
+
+```ts
+.filter(
+    and(
+        BusinessPartner.FIRST_NAME.equals('Alice'),
+        BusinessPartner.LAST_NAME.notEquals('Bob')
+    )
+)
+```
+
+Can be reduced to this:
+
+```ts
+.filter(
+    BusinessPartner.FIRST_NAME.equals('Alice'),
+    BusinessPartner.LAST_NAME.notEquals('Bob')
+)
+```
+
+More advanced filter can be found [here](#Available-Filter-Expressions)
+
+#### OrderBy
+
+Add order-by statements to the request.
+
+#### Other Set operations
+
+- `skip` skips a number of entities. Useful for paging.
+- `top` limits the number of returned entities.
```

Comment:
```suggestion
`skip` allows you to skip a number of results in the requested set. It can be useful for paging:

```ts
MyEntity.requestBuilder().
  .getAll()
  .skip(10)
```

The example above skips the first 10 results.

`top` limits the number of returned results. This can also be useful for paging:
```ts
MyEntity.requestBuilder().
  .getAll()
  .top(10)
```

The example above skips the first 10 results, that match your request.
```